src/components/MarkDown/MarkdownEditor/MarkdownEditor.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { MarkDownEditor } from './MarkDownEditor';
+import { MARKDOWN_EDITOR_TYPES } from './MarkDownEditor.constants';
+import { StoryCard } from '../../.storybook';
+
+export default {
+  title: 'Inputs/Markdown/MarkDownEditor',
+  component: MarkDownEditor
+} as ComponentMeta<typeof MarkDownEditor>;
+
+const Template: ComponentStory<typeof MarkDownEditor> = args => {
+  const [value, setValue] = useState<string>();
+  const isErroor = value === '';
+  
+  return (
+    <StoryCard isContrast isContentFull>
+      <MarkDownEditor
+        {...args}
+        text={args.text ?? value}
+        onChange={setValue}
+        error={args.error || isErroor}
+        errorText={(args.error || isErroor) ? args.errorText ?? 'Please input the text' : undefined}
```

Comment:
nit: the inner parentheses here aren't required. These again could be formatted correctly on save. Not a biggie though. I've noticed this through these files.